openedx_events/event_bus/avro/schema.py

```diff
@@ -83,6 +83,21 @@ def _create_avro_field_definition(data_key, data_type, previously_seen_types,
                 f" {set(SIMPLE_PYTHON_TYPE_TO_AVRO_MAPPING.keys())}"
             )
         field["type"] = {"type": PYTHON_TYPE_TO_AVRO_MAPPING[data_type_origin], "items": avro_type}
+    elif data_type_origin == dict:
+        # returns types of dict contents
+        # if data_type == Dict[str, int], arg_data_type = (str, int)
```

Comment:
nit: remove code comments

Reply:
I don't think that's commented code but an explanation of what arg_data_type is